lib/src/data/models/movie_page_model.dart

```diff
@@ -0,0 +1,26 @@
+import 'movie_model.dart';
+
+class MoviePageModel {
+  final int page;
+  final List<MovieModel> results;
+  final int totalPages;
+  final int totalResults;
+
+  MoviePageModel({
+    required this.page,
+    required this.results,
+    required this.totalPages,
+    required this.totalResults,
+  });
+
+  factory MoviePageModel.fromJson(Map json) {
+    return MoviePageModel(
+      page: json['page'],
+      results: MovieModel.movieList(
+        List.from(json['results']),
+      ),
```

Comment:
Here you should made the mapping of all the movies

Reply:
`List<MovieModel>.from(json['results']).map((movie) => MovieModel.fromJson(movie)).toList())`

Reply:
Right! Done